### PR TITLE
Added Color Swatch Column

### PR DESCRIPTION
**Pull request by morganchorlton3**
Added simple color swatch column for filament table

### CODE REVIEW

**Comment by RVxLab** [comment]
Related discussion: https://github.com/RVxLab/filament-colorpicker/discussions/7

**Comment by RVxLab** [comment]
Looks good code wise, I'll check it out visually later today. Thanks for the PR!

**Comment by morganchorlton3** [comment]
Brill Thanks a lot, and thanks for the help with my first contribution to an open source project :)

File: resources/views/colorswatch.blade.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div style="background-color: {{ $getState() }}; width: 3rem; height: 1.75rem; margin-left: 1rem; border-radius: 10px;">
```
Comment (RVxLab) [inline resources/views/colorswatch.blade.php:1]:
```suggestion
<div style="background-color: {{ $getState() }}; width: 1.5rem; height: 1.5rem; margin-left: 1rem; border-radius: 6px;">
```

**Review by RVxLab — CHANGES_REQUESTED** [review]
There are a couple of changes that are needed. In the current state the swatch causes a view not found error.

**Review by RVxLab — APPROVED** [review]

**Pull request merged**